CODE_OF_CONDUCT.md

```diff
@@ -6,6 +6,22 @@ The latest version of the upstream CNCF Code of Conduct is at https://github.com
 
 If the version reproduced below is out of date, the Code of Conduct which applies to the cert-manager project is the latest version upstream.
 
+## cert-manager Project Code of Conduct Committee
+
+Given the size of the team involved with cert-manager, there is no explicitly
+defined separate team of people responsible for policing the code of conduct
+for matters which fall exclusively under the [jurisdiction](https://github.com/cncf/foundation/blob/main/code-of-conduct/coc-committee-jurisdiction-policy.md) of the cert-manager project.
+
+If a committee is needed to evaluate a potential violation of the Code of Conduct,
+that team shall be made up of both the steering committee and all maintainers.
```

Comment:
```suggestion
that team shall be made up of both the steering committee and maintainers.
```

Or do we actually need to include all maintainers?

Reply:
Given how rare this should be I think it's reasonable to ask all maintainers to attend - but that said, I think you're right that it's probably too strict to say "all" here since some might not have the time to spare.

I'll rewrite it to say everyone is asked to join, and to take into account that not everyone is required.